src/oc/storage/resources/board.clj

```diff
@@ -155,6 +155,10 @@
     (delete-board! conn uuid)))
 
   ([conn :guard db-common/conn? uuid :guard #(schema/validate lib-schema/UniqueID %)]
+  ;; Delete interactions
+  (try
+    (db-common/delete-resource conn common/interaction-table-name :board-uuid uuid)
+    (catch java.lang.RuntimeException e)) ; it's OK if there are no interactions to delete
```

Comment:
Maybe it's better to check before if there are 0 interactions and don't catch a generic RuntimeException that could hide another kind of error?
Maybe we can use a more specific error if there is one.
It's not so important since it's in a delete operation.